pilot/pkg/features/pilot.go

```diff
@@ -334,4 +334,8 @@ var (
 
 	EnableEDSCaching = env.RegisterBoolVar("PILOT_ENABLE_EDS_CACHE", true,
 		"If true, Pilot will cache EDS responses.").Get()
+
+	AllowMetadataCertsInMutual = env.RegisterBoolVar("PILOT_ALLOW_METADATA_CERTS_DR_MUTUAL", false,
```

Comment:
what is the migration? explicit in all DRs?

Reply:
Yes. for folks like us who are relying on setting TLS context as MUTUAL and using metadata certs to migrate away to ISTIO_MUTUAL. Because as per API semantics, MUTUAL should have TLS certs specified in DR (not rely on metadata ). ISTIO_MUTUAL processes the default resource names or metadata certs.